src/elements/content-preview/PreviewNavigation.js

```diff
@@ -27,28 +38,45 @@ const PreviewNavigation = ({ collection = [], currentIndex, intl, onNavigateLeft
     }
 
     return (
-        <>
-            {hasLeftNavigation && (
-                <PlainButton
-                    className="bcpr-navigate-left"
-                    onClick={onNavigateLeft}
-                    title={intl.formatMessage(messages.previousFile)}
-                    type="button"
-                >
-                    <IconNavigateLeft />
-                </PlainButton>
-            )}
-            {hasRightNavigation && (
-                <PlainButton
-                    className="bcpr-navigate-right"
-                    onClick={onNavigateRight}
-                    title={intl.formatMessage(messages.nextFile)}
-                    type="button"
-                >
-                    <IconNavigateRight />
-                </PlainButton>
-            )}
-        </>
+        <NavRouter history={history}>
+            <Route path={['/:activeTab/:deeplink', '/']}>
+                {({ match }) => (
```

Comment:
I think what we can do here is extract the `history` prop from the `Route` component:

https://reacttraining.com/react-router/web/api/history

```suggestion
{({ match, history }) => (
```

That will let us use the fresh history object created by the NavRouter's MemoryRouter when the `history` prop is `undefined`